tuxemon/event/actions/modify_monster_stats.py

```diff
@@ -88,6 +92,8 @@ def start(self) -> None:
 
         monster_stats = [StatType(self.stat)] if self.stat else list(StatType)
         amount_stat = 1 if self.amount is None else self.amount
+        if amount_stat == 1 and self.rand1 and self.rand2:
```

Comment:
in the bounds? more like rand2 > rand1

Reply:
Ah right, these are the bounds ok.
Then I guess we could find better names for these identifiers, because at first I thought it was something else.

Maybe `min_stat_increase` / `max_stat_increase`.
There should at least be a `min`/`max` opposition in the names.

Reply:
Right, I misunderstood the point of these identifiers.

Then they should be rename, to emphase they are in a min/max opposition.
Maybe `min_stat_change`/`max_stat_change`.

Reply:
I used simply lower_ and upper_bond, since there is another event action using these labels

Reply:
Ok, names look better, but I don't get how the values are going to be provided...
They are never updated as far as I can see.
I guess they should be provided in the init, right?